addons/hr_recruitment/migrations/9.0.1.0/noupdate_changes.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8'?>
+<openerp>
```

Comment:
This is an automatic file generated by https://github.com/OCA/OpenUpgrade/blob/9.0/scripts/compare_noupdate_xml_records.py, so we can leave it as is for this version and change the script for next one.